features/karaf/src/main/feature/feature.xml

```diff
@@ -12,8 +12,8 @@
 <features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
 
     <feature name="openhab-runtime-base" description="openHAB Runtime Base" version="${project.version}">
-        <feature>shk-esh-base</feature>
-        <feature>shk-esh-io-console-karaf</feature>
+        <feature>esh-base</feature>
```

Comment:
Those have been added to the features where needed. I didn't add them to the runtime base yet to keep it lean.